server/app/views/applicant/ProgramIndexTemplate.html

```diff
@@ -82,6 +82,30 @@ <h2 th:text="#{title.getStartedSection}"></h2>
             th:replace="~{applicant/ProgramCardsSectionFragment :: cardsSection(${commonIntakeSection.get()})}"
           ></div>
         </div>
+        <div
```

Comment:
**note:** I'll probably put this in a fragment eventually, but am leaving it here until I've figured out the actual program filtering.  I don't want to complicate that by adding a fragment yet.